docs/source/hardware_and_software_configuration/configuring/configuring_dual_hubs/configuring-dual-hubs.rst

```diff
@@ -54,22 +39,123 @@ need the following items:
+to daisy chain the two Hubs together. Before you do this, ensure that
+neither Hub is powered on.
+
+Use the XT30 extension cable to connect an XT30 power port on the Control
+Hubs to an XT30 power port on the other Expansion Hub.
+
+<INSET IMAGE>
+
+2. The Control Hub and Expansion Hub use the RS-485 serial bus standard
+to communicate between devices. You can use the 3-pin JST PH cable to
+connect one of the ports labeled “RS485” on the Control Hub to one of
+the ports labeled “RS485” on the Expansion Hub.
+
+<INSERT IMAGE>
+
+Note that it is not important which “RS485” port that you select on the
+Expansion Hub or Control Hub. Either port should work.
+
+.. figure:: images/RS485Port.jpg
+   :align: center
+
+3. Once you have the two devices daisy chained together (12V power and
+RS-485 signal) you can connect the battery and power switch, and power
+on the devices.
+
+<INSERT IMAGE>
+
+Configuring Both devices
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you successfully daisy chained your Expansion Hub and Control Hub,
+then you should be able to create a new configuration file that includes
+both devices.
+
+**Note:** If you already have a configuration that contains just the
+Control Hub, you can add the Expansion Hub by editing the configuration
+and pressing the "Scan" button.
+
+Connect your Driver Hub to the Control Hub's WiFi network and select the
+Configure Robot option from the Driver Station app. Press the New button
+to create a new configuration file. When you first scan for hardware, your
+Robot Controller should detect the empedded Control Hub. The Robot Controller
+will automatically label this device as an Control Hub “Portal”. The Robot
+Controller will talk through this portal to the individual Hubs.
+
+<INSERT IMAGE>
+
+If you click on the Portal item in the configuration screen, you should
+see both the Control Hub and the Expansion Hub listed.
+
+<INSERT IMAGE>
+
+You can save this configuration file and return to the main screen of
+the Driver Station. After the robot has been restarted, both Hubs should
+have a solid green LED that blinks blue every ~5 seconds.
+
+Congratulations, you are now ready to use your combination of Control and
+Expansion Hubs! You can configure and operate these Hubs as you would an
+individual Hub.
+
+Using Two Expansion Hubs
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Teams without access to a Control Hub may use two Expansion Hubs on
+their robot. This is not recommended as it creates extra steps in
+order to control the robot.
+
+Additional Equipment Needed
+---------------------------
+
+There is some additional equipment required for teams who aren't
+using a Control Hub on their robot.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+   :class: longtable
+
+   * - Required Item(s)
+     - Image
+
+   * - A FIRST-approved Android smartphone with the FTC Robot Controller
+       app installed. For a list of FIRST-approved Android smartphones,
+       refer to the current Competition Manual, rule R704.
```

Comment:
Rephrase to not point to s specific rule number. Rule numbers may shift year to year.  

Suggest something like:

> Check the Robot Construction rules in the [Competition Manual](https://ftc-resources.firstinspires.org/file/ftc/game/manual) for details.